openmicroscopy/openmicroscopy
# Get thumbnails

# What this PR does

- fix getThumbnailsByLongestSideSet in readonly group https://trello.com/c/3eEBEPG4/7-retrieving-multiple-thumbnails
- allow loading thumbnails in a batches


# Testing this PR

- test thumbnail version like in https://github.com/openmicroscopy/openmicroscopy/pull/3713#issuecomment-95146309
- load plate, all thumbnail should be loaded in a batches
- log in and test if updating rendeting update thumbnail:
  - test centre panel, search and plate grid
- consider any usecases I could miss! :)

# TODO:

- [x]  review pixel vs image in a args (looks like gateway API is not consistent)
- [x] **performance testing** That should happen on real production deployment (with and without nginx micro cache turned on)

cc @pwalczysko 

----
--depends-on https://github.com/openmicroscopy/openmicroscopy/pull/5173

--rebased-to https://github.com/openmicroscopy/openmicroscopy/pull/5081 ~~https://github.com/openmicroscopy/openmicroscopy/pull/5112~~ ~~https://github.com/openmicroscopy/openmicroscopy/pull/5138~~ #5158
